asab/library/service.py

```diff
@@ -287,6 +287,9 @@ async def _read_disabled(self):
 				self.Disabled = yaml.safe_load(disabled)
 				if self.Disabled is None:
 					self.Disabled = {}
+				elif isinstance(self.Disabled, set):
+					# This is for a backward compatibility (Aug 2023)
```

Comment:
If this is only for the backward compatibility, shouldn't there be a message like: _"Please use dictionary instead of set"_?

Reply:
Yes, it could be.